src/app/public/src/modules/sidebar/sidebar-wrapper.component.html

```diff
@@ -0,0 +1,14 @@
+<div class="stache-sidebar-wrapper" [ngClass]="{'sidebar-closed': sidebarClosed, 'sidebar-open': !sidebarClosed  }">
+  <div class="stache-sidebar-header">
+    <button *ngIf="!sidebarClosed" class="sky-btn sky-btn-link-inline stache-sidebar-button" (click)="closeSidebar()" aria-label="sidebar close">
+      <i class="fa fa-chevron-left" aria-hidden="Close"></i>
+    </button>
+    <button *ngIf="sidebarClosed" class="sky-btn sky-btn-link-inline stache-sidebar-button" (click)="openSidebar()" aria-label="sidebar open">
+      <i class="fa fa-chevron-right" aria-hidden="Open"></i>
```

Comment:
Same here, regarding `aria-hidden`. 

Also, I think you should use the "hamburger" icon to signify a menu.

Reply:
"Hamburger" I leaned away from, as the omnibar also uses the hamburger icon for it's mobile sizes.  Having the two stacked so close seemed strange.  Using the [chevrons](https://developer.blackbaud.com/skyux/design/styles/icons) to make it more of a panel seemed like a way around this. Willing to re-evaluate, use an x / hamburger?